pkg/controller/installation/windows_controller.go

```diff
@@ -151,6 +151,12 @@ func AddWindowsController(mgr manager.Manager, opts options.AddOptions) error {
 		return fmt.Errorf("tigera-windows-controller failed to watch FelixConfiguration resource: %w", err)
 	}
 
+	// Watch for changes to IPAMConfiguration.
+	err = c.Watch(&source.Kind{Type: &apiv3.IPAMConfiguration{}}, &handler.EnqueueRequestForObject{})
```

Comment:
Any time you're watching for a projectcalico.org/v3 resource, you need to use a special pattern because the watch won't establish until the apiserver is running.

Here is an example of starting a watch: https://github.com/tigera/operator/blob/0c1c706b30dc8ab66247bdc911d005a71b4283ee/pkg/controller/manager/manager_controller.go#L99-L104

And then your controller needs to wait for the Watch to establish before performing a Get() on that resource, like this: https://github.com/tigera/operator/blob/master/pkg/controller/manager/manager_controller.go#L296-L300

Reply:
Got it, thanks @caseydavenport! Done